### ISSUE TITLE
Auto-GPT not able to provide me with any research material. I am a noob. Can someone please help with the goals or prompts I need to enter. I have been struggling for almost 2 weeks now and have gotten no results

### ISSUE THREAD

**Issue by Chcornel411** [Significant-Gravitas/Auto-GPT#3100]
### ⚠️ Search for existing issues first ⚠️

- [X] I have searched the existing issues, and there is no existing issue for my problem

### Which Operating System are you using?

Windows

### Which version of Auto-GPT are you using?

Latest Release

### GPT-3 or GPT-4?

GPT-3.5

### Current behavior 😯

s

### Your prompt 📝

```yaml
# Paste your prompt here
```

### Your Logs 📒

```log
<insert your logs here>
```

**Comment by ozzymuppet** [comment]
Same. I've been trying to get it to search for certain products on eBay, and dump me a list in CSV . I give it clear instructions, specific URLs, etc. It will come up with a decent plan, then use the data it gets poorly, and come up with a worse and worse plan until it's pointless. Right now I'm not convinced it can actually do anything.

**Comment by themetalleg** [comment]
Be aware, that this is not a finished software. This is still release 0.2.2, not 1.something.
At the moment you can get an idea, what later will be possible.

**Comment by ozzymuppet** [comment]
Yeah, true. No disrespect intended to developers, it's awesome. In a couple of months it's going to be wild!

**Comment by Chcornel411** [comment]
Now that is rude 

⁣Charlton Cornelius
GISP
081 7382 790 (c)
***@***.***



Get BlueMail for Android ​
